app/src/core/components/FileList.tsx

```diff
@@ -0,0 +1,107 @@
+import { Box, List, ListItem, ListItemIcon, ListItemText, Typography, LinearProgress } from '@mui/material';
+
+import { InsertDriveFile } from '@mui/icons-material';
+import styled from 'styled-components';
+
+const FileListContainer = styled(List)`
+    width: 100%;
+    margin-top: ${({ theme }) => theme.spacing(2)};
+`;
+
+const FileListItem = styled(ListItem)`
+    display: flex;
+    border: 1px solid ${({ theme }) => theme.palette.divider};
+    border-radius: ${({ theme }) => theme.shape.borderRadius}px;
+    margin-bottom: ${({ theme }) => theme.spacing(2)};
+`;
+
+const FileListImage = styled.img`
+    width: 40px;
+    height: 40px;
```

Comment:
nit: ideally we'd use `theme` sizing values for stuff like this, for easier customization across projects, but if we cant, imo arbitrary exact values should use `rem` units except for precision pixel placements, so we get screen zooming/scaling support for free.